ui/src/constants/index.js

```diff
@@ -1,108 +1,204 @@
 const DATA_URL = `https://virustrack.live/site-data`
 
 const COUNTRIES = [
-    "!Global",
-    "!Outside Mainland China",
-    "Afghanistan",
-    "Algeria",
-    "Andorra",
-    "Argentina",
-    "Armenia",
-    "Australia",
-    "Austria",
-    "Azerbaijan",
-    "Bahrain",
-    "Belarus",
-    "Belgium",
-    "Bhutan",
-    "Bosnia and Herzegovina",
-    "Brazil",
-    "Cambodia",
-    "Cameroon",
-    "Canada",
-    "Chile",
-    "Colombia",
-    "Costa Rica",
-    "Croatia",
-    "Czech Republic",
-    "Denmark",
-    "Dominican Republic",
-    "Ecuador",
-    "Egypt",
-    "Estonia",
-    "Faroe Islands",
-    "Finland",
-    "France",
-    "Georgia",
-    "Germany",
-    "Gibraltar",
-    "Greece",
-    "Hong Kong",
-    "Hungary",
-    "Iceland",
-    "India",
-    "Indonesia",
-    "Iran",
-    "Iraq",
-    "Ireland",
-    "Israel",
-    "Italy",
-    "Japan",
-    "Jordan",
-    "Kuwait",
-    "Latvia",
-    "Lebanon",
-    "Liechtenstein",
-    "Lithuania",
-    "Luxembourg",
-    "Macau",
-    "Mainland China",
-    "Malaysia",
-    "Mexico",
-    "Monaco",
-    "Morocco",
-    "Nepal",
-    "Netherlands",
-    "New Zealand",
-    "Nigeria",
-    "North Macedonia",
-    "Norway",
-    "Oman",
-    "Others",
-    "Pakistan",
-    "Palestine",
-    "Peru",
-    "Philippines",
-    "Poland",
-    "Portugal",
-    "Qatar",
-    "Romania",
-    "Russia",
-    "Saint Barthelemy",
-    "San Marino",
-    "Saudi Arabia",
-    "Senegal",
-    "Serbia",
-    "Singapore",
-    "Slovakia",
-    "Slovenia",
-    "South Africa",
-    "South Korea",
-    "Spain",
-    "Sri Lanka",
-    "Sweden",
-    "Switzerland",
-    "Taiwan",
-    "Thailand",
-    "Togo",
-    "Tunisia",
-    "UK",
-    "US",
-    "Ukraine",
-    "United Arab Emirates",
-    "Vatican City",
-    "Vietnam"
-  ]
+  "!Global",
```

Comment:
We're at the mercy of incompatible country names here -- that was one of the delays yesterday.  Rather than defining these in a constant, a future version may use JSON that you load from the server/database.  That way we have homologized naming.  See the `covidvu.vudpatch` module if you are curious about some of the heuristics I had to code to overcome disparate names for countries, states, and regions.